src/applications/disability-benefits/all-claims/tests/pages/form0781/workflowChoicePage.unit.spec.js

```diff
@@ -1,12 +1,212 @@
+        definitions={{}}
+        data={{
+          newDisabilities: [
+            {
+              condition: 'ankle replacement (ankle arthroplasty), bilateral',
+            },
+            {
+              condition: 'somatic symptom disorder (SSD)',
+            },
+            {
+              condition: 'varicocele, left',
+            },
+          ],
+        }}
+      />,
+    );
+
+    const conditionsParagraph = getByText(
+      /You selected these new conditions for your disability claim:/,
+    );
+
+    const listElement = within(conditionsParagraph.parentElement).getByRole(
+      'list',
+    );
+
+    within(listElement).getByText(
+      'Ankle replacement (ankle arthroplasty), bilateral',
+    );
+
+    within(listElement).getByText('Somatic symptom disorder (SSD)');
+
+    within(listElement).getByText('Varicocele, left');
+  });
+
+  // This situation shouldn't occur but we should be defensive
+  it('does not display a conditions list if no new conditions exist', () => {
+    const { queryByText } = render(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        definitions={{}}
+        data={{}}
+      />,
+    );
+
+    expect(
+      queryByText(
+        /You selected these new conditions for your disability claim:\s/i,
+      ),
+    ).not.to.exist;
+  });
+
+  it('Displays a selection of choices on filling out 0781 and incplues examples', () => {
+    const onSubmit = Sinon.spy();
+    const { container, getByText } = render(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{}}
+        definitions={{}}
+        formData={{}}
+        onSubmit={onSubmit}
+      />,
+    );
+
+    getByText('Statement about mental health conditions (VA Form 21-0781)');
+
+    const radioButtons = $$('va-radio');
+    expect(radioButtons.length).to.equal(1);
+
+    expect(
+      container.querySelector(
+        `va-radio-option[label="${
+          form0781WorkflowChoiceLabels[
+            form0781WorkflowChoices.COMPLETE_ONLINE_FORM
+          ]
+        }"]`,
+        container,
+      ),
+    ).to.exist;
+
+    expect(
+      container.querySelector(
+        `va-radio-option[label="${
+          form0781WorkflowChoiceLabels[
+            form0781WorkflowChoices.SUBMIT_PAPER_FORM
+          ]
+        }"]`,
+        container,
+      ),
+    ).to.exist;
+
+    expect(
+      container.querySelector(
+        `va-radio-option[label="${
+          form0781WorkflowChoiceLabels[
+            form0781WorkflowChoices.OPT_OUT_OF_FORM0781
+          ]
+        }"]`,
+        container,
+      ),
+    ).to.exist;
+
+    const addlInfo = container.querySelector('va-accordion-item');
+    const headline = addlInfo.querySelector('h3[slot="headline"]');
+    expect(headline).to.have.text(
+      'Examples of mental health conditions and traumatic events',
+    );
+  });
+
+  it('should prevent continuing if a selection is not made', () => {
```

Comment:
Is an error message displayed if attempting to continue from this page without making a selection? Should the content of that error message be tested?

Reply:
Yeah this is a good call, figured this out in https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/5475dff3a6c59f00f7e8606a7ecef76ee44439a9